tests/sentry/integrations/slack/webhooks/actions/test_status.py

```diff
@@ -870,91 +862,6 @@ def test_assign_issue_user_has_identity_block_kit_through_unfurl(self):
         )
         assert resp.data["blocks"][2]["text"]["text"].endswith(expect_status), resp.data["text"]
 
-    def test_response_differs_on_bot_message(self):
```

Comment:
k sorry i just moved this down because it was weird it was just in the middle of a bunch of assignment tests